packages/nc-gui-v2/utils/validation.ts

```diff
@@ -127,3 +127,17 @@ export const importExcelUrlValidator = {
     })
   },
 }
+
+export const extraParameterValidator = {
+  validator: (rule: any, value: any, callback: (errMsg?: string) => void) => {
+    for (const param of value) {
+      if (param.key === '') {
+        callback('Parameter key cannot be empty')
```

Comment:
Use of callback is deprecated, please use Promise.reject instead (otherwise it throws warnings in the console)